src/app/work-item/work-item-detail/work-item-detail.component.ts

```diff
@@ -614,6 +647,44 @@ export class WorkItemDetailComponent implements OnInit, AfterViewInit, OnDestroy
     }
   }
 
+  createComment(comment) {
+    this.workItemService
+        .createComment(this.workItem.relationships.comments.links.related, comment)
+        .subscribe((comment) => {
+          console.log(this.loggedInUser);
+            comment.relationships['created-by'].data = this.loggedInUser;
+            this.workItem.relationships.comments.data.splice(0, 0, comment);
```

Comment:
As now I'm passing "comments" itself as the inputs (check details comp), it does.